NICE.Identity.Management/ClientApp/src/components/FilterBox/FilterBox.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+
+import {
+	FilterPanel,
+	FilterGroup,
+	FilterOption,
+} from "@nice-digital/nds-filters";
+
+import "@nice-digital/nds-filters/scss/filters.scss";
+import { capitaliseWord } from "../../helpers/capitaliseWord";
+
+type FiltersType = {
+	name: string;
+	value: string;
+};
+
+type FilterBoxProps = {
+	name: string;
+	filters: Array<FiltersType> | Array<string>;
+	selected: Array<string>;
+	onCheckboxChange: (e: string) => void;
+};
+
+export const FilterBox = (props: FilterBoxProps): React.ReactElement => {
+	let filters: any = [];
```

Comment:
any - what's this an array of?

Reply:
I struggled getting this to work as it can be populated by a variable which is either an array of FiltersType objects or an array of strings. I think I've managed to overcome it.